src/hotspot/os/windows/pdh_interface.cpp

```diff
@@ -111,10 +111,10 @@ bool PdhDll::PdhAttach(void) {
   }
   while (InterlockedCompareExchange(&_critical_section, 0, 1) == 0);
   return (_PdhAddCounter != NULL && _PdhOpenQuery != NULL
-         && _PdhCloseQuery != NULL && PdhCollectQueryData != NULL
+         && _PdhCloseQuery != NULL && _PdhCollectQueryData != NULL
          && _PdhGetFormattedCounterValue != NULL && _PdhEnumObjectItems != NULL
-         && _PdhRemoveCounter != NULL && PdhLookupPerfNameByIndex != NULL
```

Comment:
Shouldn't this also be checking the underscore version ie `_PdhLookupPerfNameByIndex`?

Reply:
Ah, oh dear, I have no idea how I missed that. Will fix as requested